image/templates/helm/stackrox-central/templates/99-openshift-monitoring.yaml

```diff
@@ -77,24 +77,43 @@ spec:
     matchNames:
       - "{{ .Release.Namespace }}"
 
+
+---
+
+apiVersion: authorization.openshift.io/v1
+kind: ClusterRole
+metadata:
+  name: cluster-extension-apiserver-authentication-reader
```

Comment:
Does Helm handle this correctly if you install both Central and Sensor charts? Or will there be a collision?

Reply:
I missed that; you commented between me pushing my changes. I will go ahead and resolve this.